source/blender/windowmanager/WM_types.h

```diff
@@ -308,6 +308,7 @@ typedef struct wmNotifier {
 /* Changes to asset data in the current .blend. */
 #define NC_ASSET (26 << 24)
 #define NC_LOGIC (27 << 24)
+#define NC_GAME (28 << 24)
```

Comment:
It is not called anywhere in the PR

Reply:
Yeah, it looks like we don't need to define a new node category for that. I'll remove it when I update the PR.